load_testing/common_flows/flow_sign_in.py

```diff
@@ -0,0 +1,68 @@
+from faker import Faker
+from random import randint
+import os
+from .flow_helper import (
```

Comment:
My editor complains about these imports, but they work 🤷‍♀️

Reply:
Your editor is wrong and your relative import is right.